src/hyperdrive-cli/commands/stakewise/validator/get-signed-exit-messages.go

```diff
@@ -32,18 +34,27 @@ func getSignedExitMessages(c *cli.Context) error {
 	// Get the client
 	sw := client.NewStakewiseClientFromCtx(c)
 
-	activeValidatorResponse, err := sw.Api.Status.GetActiveValidators()
+	activeValidatorResponse, err := sw.Api.Status.GetValidatorStatuses()
 	if err != nil {
 		return fmt.Errorf("error while getting active validators: %w", err)
 	}
+	var activeValidators []string // beacon.ValidatorPubkey.HexWithPrefix()
+	for pubKey, status := range activeValidatorResponse.Data.BeaconStatus {
+		if status == swapi.ActiveOngoing || status == swapi.ActiveSlashed {
```

Comment:
`swapi.ActiveSlashed` shouldn't be included here; if someone's slashed, any voluntary exits they submit are going to be ignored so there's no point in being able to generate them.